backend/tasks/create-winter-seasons/create-winter-seasons.js

```diff
@@ -0,0 +1,329 @@
+// This script creates winter seasons for parks that have hasWinterFeeDates = true.
+// It will skip creating a season if one already exists for the given operating year and publishable.
+// Also removes Winter fee DateRanges from regular seasons.
+
+import "../../env.js";
+
+import {
+  Dateable,
+  DateRange,
+  DateType,
+  Park,
+  Publishable,
+  Season,
+} from "../../models/index.js";
+import * as STATUS from "../../constants/seasonStatus.js";
+
+// Run all queries in a transaction
+const transaction = await Season.sequelize.transaction();
+
+if (process.env.SEQUELIZE_LOGGING === "false") {
+  console.warn(
+    "SEQUELIZE_LOGGING is set to false. No SQL queries will be logged.",
+  );
+
+  Season.sequelize.options.logging = false;
+}
+
+// Print errors and roll back transaction on exceptions
+process.on("uncaughtException", (err) => {
+  console.error(`\n${err.message}\n`);
+  transaction?.rollback();
+  throw new Error(err);
+});
+
+// Get the operating year from command line arguments
+const operatingYear = Number(process.argv[2]);
+
+if (isNaN(operatingYear)) {
+  console.info("Usage example: npm run create-winter-seasons 2027");
+  throw new Error("Missing operating year");
+}
+
+// Track the number of rows inserted/deleted
+let publishablesAdded = 0;
+let dateablesAdded = 0;
+let winterSeasonsAdded = 0;
+let winterDateRangesAdded = 0;
+let winterDateRangesDeleted = 0;
+
+/**
+ * Creates a new Publishable or Dateable ID and associates it with the given record, if it doesn't already have one.
+ * @param {Park} record The record to check and update
+ * @param {string} keyName The name of the key to check ("publishableId" or "dateableId")
+ * @param {any} KeyModel Db model to use for creating the new ID (Publishable or Dateable)
+ * @returns {Promise<{key: number, added: boolean}>} The ID and whether it was created
+ */
+async function createKey(record, keyName, KeyModel) {
```

Comment:
`createPublishable`, `createKey` and `createDateable` are all repeated from the create-season.js script. Should we move them to a shared library file?  

Also, `create-season.js` is very similar to this script but it's the only script without a folder or a README. It wouldn't hurt to standardize this as well.